projects/js-packages/videopress-core/composer.json

```
@@ -0,0 +1,32 @@
+{
+	"name": "automattic/jetpack-videopress-core",
+	"description": "VideoPress Core Functionality",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"test-js": [
+			"pnpm run test"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true
```

I created the mirror repo, so we now can add this to our project:

```suggestion
	"prefer-stable": true,
	"extra": {
		"autotagger": true,
		"npmjs-autopublish": true,
		"mirror-repo": "Automattic/jetpack-videopress-core",
		"changelogger": {
			"link-template": "https://github.com/Automattic/jetpack-videopress-core/compare/${old}...${new}"
		}
	}
```

---

Thanks, done!